lib/main.js

```diff
@@ -423,7 +327,10 @@ const Hydrogen = {
       if (moveDown === true) {
         codeManager.moveDown(editor, row);
       }
-      this.createResultBubble(editor, { code, row, cellType });
+      this.checkForKernel(store, kernel => {
+        //$FlowFixMe
+        result.createResult(store, { code, row, cellType });
```

Comment:
I can't fix this flow error for the life of me. It is also in 3 other places

Reply:
Will try to fix this in next week or so.

Reply:
Fixed in a1877f2 :)